content/module-reference/utility-modules/shared/collection-filters.md

```diff
@@ -0,0 +1,141 @@
+
+## color labels
+
+The following image shows the color labels widget, set to filter images having yellow or green color labels:
+
+![color filter](./collection-filters/color-filter.png#w33)
+
+You can interact with this filter widget as follows:
+
+- Click on a color label to include images with that label
+- Ctrl+click on a color label to include images without that label
+- Click or Ctrl+click on the gray icon to act on all color labels simultaneously
+- Use the last button to define how to handle the selection of multiple color labels. Select ![and](./collection-filters/color-filter-and.png#icon) (and) to filter images having _all_ of the selected color labels; Select ![or](./collection-filters/color-filter-or.png#icon) (or) to filter images with _at least one_ of the selected color labels.
+
+## rating
+
+This is the classic rating selection widget that used to be shown in darktable's top panel by default.
+
+![rating filter](./collection-filters/rating-filter.png#w33)
+
+It is composed of a pair of comboboxes. The first combobox is used to choose the operator to use and (if applicable) the second combobox selects the number of stars to be applied to that operator. If you choose "All", the second combobox will not appear.
+
+## range rating
+
+This new widget also allows you to select images by star rating, this time using a new "range" widget. The following image shows the widget with ratings of 2-4 stars selected.
+
+![range rating filter](./collection-filters/range-rating-filter.png#w33)
+
+You can choose a new range of ratings to filter by clicking and dragging over the widget's interface. You can also access pre-defined ranges by right-clicking and selecting from the popup menu. The number of applicable images is also shown against each entry in this menu.
+
+## range filters for numeric attributes
+
+Numeric attributes (aperture, exposure, focal length, iso, aspect ratio) are filtered using a widget that shows a histogram of the number of images available depending on the value of the given attribute (similar to the [_timeline_](../lighttable/timeline.md) in the lighttable view).
+
+For example, the following image shows the widget when used for selecting by aperture:
+
+![aperture range filter](./collection-filters/aperture-range-filter.png#w33)
+
+As with the range rating filter, you can select a range of values to filter by clicking and dragging on the widget. You can also access predefined ranges by right-clicking and selecting from the popup menu. The number of applicable images is also shown against each entry in this menu.
+
+You can also use the "min" and "max" text entry fields in the main module interface to manually define the bounds of the selection.
+
+## date attributes
+
+As with numeric and rating range filters, date/time filters are represented using a "range" widget. Select a range of values by clicking and dragging on the widget and use the right-click menu for more options.
```

Comment:
**You can** select a range of values by clicking and dragging on the widget and use the right-click menu for more options.
(I would say it's more in line with the rest of the sentences)...